Mage.Sets/src/mage/cards/t/TzaangorShaman.java

```diff
@@ -0,0 +1,90 @@
+package mage.cards.t;
+
+import mage.MageInt;
+import mage.abilities.DelayedTriggeredAbility;
+import mage.abilities.common.DealsCombatDamageToAPlayerTriggeredAbility;
+import mage.abilities.effects.common.CopyTargetSpellEffect;
+import mage.abilities.effects.common.CreateDelayedTriggeredAbilityEffect;
+import mage.abilities.keyword.FlyingAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.Duration;
+import mage.constants.SubType;
+import mage.game.Game;
+import mage.game.events.GameEvent;
+import mage.game.stack.Spell;
+import mage.target.targetpointer.FixedTarget;
+
+import java.util.UUID;
+
+/**
+ * @author PurpleCrowbar
+ */
+public final class TzaangorShaman extends CardImpl {
+
+    public TzaangorShaman(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{2}{U}{R}");
+        this.subtype.add(SubType.MUTANT, SubType.SHAMAN);
+        this.power = new MageInt(3);
+        this.toughness = new MageInt(3);
+
+        // Flying
+        this.addAbility(FlyingAbility.getInstance());
+
+        // Sorcerous Elixir — Whenever Tzaangor Shaman deals combat damage to a player, copy the next instant or
+        // sorcery spell you cast this turn when you cast it. You may choose new targets for the copy.
+        this.addAbility(new DealsCombatDamageToAPlayerTriggeredAbility(new CreateDelayedTriggeredAbilityEffect(
+                new TzaangorShamanAbility()).setText(" copy the next instant or sorcery spell you cast this turn when you cast it. You may choose new targets for the copy."), false).withFlavorWord("Sorcerous Elixir")
+        );
+    }
+
+    private TzaangorShaman(final TzaangorShaman card) {
+        super(card);
+    }
+
+    @Override
+    public TzaangorShaman copy() {
+        return new TzaangorShaman(this);
+    }
+}
+
+class TzaangorShamanAbility extends DelayedTriggeredAbility {
```

Comment:
why doesn't this use `CopyNextSpellDelayedTriggeredAbility`?

Reply:
Fixed, was originally based off of Sea Gate Stormcaller's code